src/beman/exemplar/CMakeLists.txt

```diff
@@ -9,8 +11,9 @@ target_sources(
     beman.exemplar
     PUBLIC
         FILE_SET HEADERS
-        BASE_DIRS ${PROJECT_SOURCE_DIR}/include
-        FILES ${PROJECT_SOURCE_DIR}/include/beman/exemplar/identity.hpp
+        BASE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../../../include
+        FILES
+            ${CMAKE_CURRENT_SOURCE_DIR}/../../../include/beman/exemplar/identity.hpp
```

Comment:
I dislike the connection at a distance this creates, but I don't have a really good solution without a lot more of reorganization. 

For optional the header file set is set up in the `include/beman/optional` directory. 

https://github.com/steve-downey/optional/blob/960da87ae54d817bc6f46fe9fda776b21f88e939/include/beman/optional/CMakeLists.txt#L7

Putting on my list to discuss over lunch in Aspen.